httpbin/httpbin.go

```diff
@@ -179,3 +185,38 @@ func (h *HTTPBin) Handler() http.Handler {
 
 	return handler
 }
+
+func (h *HTTPBin) HeadersProcessor(headers http.Header) http.Header {
+	result := make(http.Header)
+	for k, v := range headers {
+		lowerCase := strings.ToLower(k)
+		skipped := false
+		for _, pattern := range h.excludeHeadersPatterns {
+			matched := pattern.Match([]byte(lowerCase))
+			if matched {
+				skipped = true
+				break
+			}
+		}
+		if skipped {
+			continue
+		}
+		result[k] = v
+	}
+
+	return result
+}
+
+func (h *HTTPBin) SetExcludeHeaders(excludeHeaders string) {
+	// comma separated list of headers to exclude from response
+	tmp := strings.Split(excludeHeaders, ",")
+	result := make([]*regexp.Regexp, len(tmp))
+
+	for i, v := range tmp {
+		pattern := wildCardToRegexp(strings.TrimSpace(strings.ToLower(v)))
+		compiled, _ := regexp.Compile(pattern)
```

Comment:
Should we throw error if compiled regex failed?

Reply:
In this case, I'd be in favor of just using `regexp.MustCompile` and panicking on error.  I think it's fine for the process to fail loudly and obviously on initial startup if misconfigured.